app/assets/javascripts/sample_types.js

```diff
@@ -96,6 +96,11 @@ var SampleTypes = {
         else {
             seek_sample_element.hide();
         }
+    },
+
+    expandIRI: function () {
+        var termIRI = $j(this).next('.term-iri');
+        termIRI.toggleClass('visible');
```

Comment:
might be better to use `.show()` from jquery, as this is what we use elsewhere and is a bit clearer

Reply:
![image](https://github.com/user-attachments/assets/e9a05307-341b-4516-aa81-dfd554a596af)
if using `.show()` , it will show the term-iri in the next row.

Reply:
OK, can leave it then and is not that important. I also realise now that it is toggling it rather than just showing it.